keep/api/models/db/migrations/versions/2024-09-22-14-16_01ebe17218c0.py

```diff
@@ -0,0 +1,62 @@
+"""Topology applications
+
+Revision ID: 01ebe17218c0
+Revises: 5d7ae55efc6a
+Create Date: 2024-09-22 14:16:17.078591
+
+"""
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "01ebe17218c0"
+down_revision = "5d7ae55efc6a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "topologyapplication",
+        sa.Column("tenant_id", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("id", sqlmodel.sql.sqltypes.GUID(), nullable=False),
+        sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("description", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.ForeignKeyConstraint(
```

Comment:
Isn't is a typo? Looks strange, 2 times same reference

Reply:
These are two different ref, or am I missing something?
```
  ["tenant_id"],
  ["tenant.id"],
```